ports/nrf/mphalport.h

```diff
@@ -43,6 +44,9 @@ typedef enum
 
 extern const unsigned char mp_hal_status_to_errno_table[4];
 
+extern int mp_interrupt_char;
```

Comment:
Please include `shared/runtime/interrupt_char.h` instead of declaring this variable.